src/applications/mhv-landing-page/tests/e2e/alerts/no-mhv-account.cypress.spec.js

```diff
@@ -0,0 +1,39 @@
+import { appName } from '../../../manifest.json';
+import ApiInitializer from '../utilities/ApiInitializer';
+import LandingPage from '../pages/LandingPage';
+import MhvRegistrationAlert from '../../../components/MhvRegistrationAlert';
+import { resolveLandingPageLinks } from '../../../utilities/data';
+
+describe(`${appName} - MHV Registration Alert - `, () => {
+  const pageLinks = resolveLandingPageLinks(false, [], 'arialLabel', false);
+
+  beforeEach(() => {
+    ApiInitializer.initializeFeatureToggle.withCurrentFeatures();
+  });
+
+  it(`alert shown for user without MHV account`, () => {
+    LandingPage.visit({ mhvAccountState: 'NONE' });
+    cy.injectAxeThenAxeCheck();
+    cy.findByRole('heading', {
+      name: MhvRegistrationAlert.defaultProps.headline,
+    }).should.exist;
+
+    // Check the cards are visible
+    pageLinks.cards.forEach(card => {
+      LandingPage.validateLinkGroup(card.title, card.links.length);
+    });
```

Comment:
Is it necessary to validate the cards _and_ links, here? If you wanted to check that the cards are rendered, throw a testid on `<CardLayout />` and check that it rendered with `cy.findByTestId()/cy.getByTestId()`. I believe we already have multiple tests for rendering links.

Reply:
I prefer the full test since I found an issue with one conditional heading being incorrect. There is no existing test specific to not having an MHV account, so I thought of it as necessary.

Reply:
So, this became a huge problem as soon as the toggles changed as the links passed into the validation code does not handle the multiple toggles and different implementations. I had to follow your suggestion and the links themselves will have to be tested in another test.